tests/Utils.ts

```diff
@@ -230,6 +231,122 @@ export class TestStoa extends Stoa
     }
 }
 
+/**
+ * This is a client for testing.
+ * Test codes can easily access error messages received from the server.
+ */
+export class TestClient
+{
+    private client: AxiosInstance;
+
+    constructor ()
+    {
+        this.client = axios.create();
+    }
+
+    public get (url: string, config?: AxiosRequestConfig): Promise<AxiosResponse>
+    {
+        return new Promise<AxiosResponse>((resolve, reject) =>
+        {
+            this.client.get(url, config)
+                .then((response: AxiosResponse) =>
+                {
+                    resolve(response);
+                })
+                .catch((reason: any) =>
+                {
+                    reject(this.handleNetworkError(reason));
+                });
+        });
+    }
+
+    public delete (url: string, config?: AxiosRequestConfig): Promise<AxiosResponse>
+    {
+        return new Promise<AxiosResponse>((resolve, reject) =>
+        {
+            this.client.delete(url, config)
+                .then((response: AxiosResponse) =>
+                {
+                    resolve(response);
+                })
+                .catch((reason: any) =>
+                {
+                    reject(this.handleNetworkError(reason));
+                });
+        });
+    }
+
+    public post (url: string, data?: any, config?: AxiosRequestConfig): Promise<AxiosResponse>
+    {
+        return new Promise<AxiosResponse>((resolve, reject) =>
+        {
+            this.client.post(url, data, config)
+                .then((response: AxiosResponse) =>
+                {
+                    resolve(response);
+                })
+                .catch((reason: any) =>
+                {
+                    reject(this.handleNetworkError(reason));
+                });
+        });
+    }
+
+    public put (url: string, data?: any, config?: AxiosRequestConfig): Promise<AxiosResponse>
+    {
+        return new Promise<AxiosResponse>((resolve, reject) =>
+        {
+            this.client.put(url, data, config)
+                .then((response: AxiosResponse) =>
+                {
+                    resolve(response);
+                })
+                .catch((reason: any) =>
+                {
+                    reject(this.handleNetworkError(reason));
+                });
+        });
+    }
+
+    private handleNetworkError (error: any): Error
```

Comment:
https://github.com/bpfkorea/boa-sdk-ts/blob/571f33d784f06e1a72b4b0226caf53c19f5f5f71/src/modules/net/error/ErrorTypes.ts#L92

I think you can import it from SDK and use it.

Reply:
I changed